shared/cryptoutil/cryptoUtil.ts

```diff
@@ -190,3 +196,9 @@ export function hmacSHA(encryptionKey: string, data: string) {
   const resultBytes = hmac.digest().getBytes().toString();
   return resultBytes;
 }
+
+export async function compressData(fileName: string): Promise<string> {
```

Comment:
Can the fn prototype be changed?

Giving some examples from unix commands

```
$ tar <flags> backupfile.<extension> <source-files>

$ cp <src> <dst>
```

here's a suggestion, I personally don't like `archiveOutfilePath` name I've come up with

`fn compress(srcPath string, archiveOutfilePath string)`

passing both dependents separately will help later on with testing or extending this

Reply:
@vharsh Can you elaborate and explain please?